oak-core/src/main/java/org/apache/jackrabbit/oak/query/QueryEngineSettings.java

```diff
@@ -182,6 +186,14 @@ public String getQueryValidatorJson() {
     public QueryValidator getQueryValidator() {
         return queryValidator;
     }
+    
+    public void setIgnoredClassNamesinCallTrace(String[] packageNames) {
+        classNamesIgnoredInCallTrace = packageNames;
```

Comment:
with the annotation you can avoid having to check for null param here.... which would overwrite the default empty-string value..... i like the code in this form and i would avoid redundant checks for null by adding the annotation.